### PR TITLE
added link to "options" sub-page

### PR DESCRIPTION
**Pull request by 62mkv**

### CODE REVIEW

**Comment by 62mkv** [comment]
closes #11732

**Comment by 62mkv** [comment]
@MathieuAA would it make sense to also re-structure "Options" page, to display entity-specific options separately from field-specific ones? not sure how big is an overlap

**Comment by MathieuAA** [comment]
What do you mean by field-specific options? Are you referring to the validations?

File: pages/jdl/entities.md [hunk 1]
```diff
@@ -296,3 +296,5 @@ Common databases:
     <td><dfn>required, minbytes, maxbytes, unique</dfn></td>
   </tr>
 </table>
+
+[Options]: ./options.html "Options"
```
Comment (MathieuAA) [inline pages/jdl/entities.md:300]: Simpler: "[Options]: options "Options"

**Comment by 62mkv** [comment]
nevermind, I noticed that the title above the table says that those specifically *entity* options, so it's fine

**Comment by 62mkv** [comment]
anything else here?

**Pull request merged**